docs/wire_compiler.md

```diff
@@ -86,6 +86,7 @@ source directory for the project so the generated sources are compiled by the Ko
 
 Replace `kotlin` with `java` to generate the Java sources instead.
 
+When `kotlin` or `java` is omitted from the Wire configuration, no files will be generated. If one of these is included, and no `sourcePath` declarations are made, this directory will be assumed to be the default directory to generate sources. 
```

Comment:
If no target is set, the plugin with throw, so the first sentence can be omitted. I like the rest, I'll add it to Wire and ping you

Reply:
thank you 👍 

adding details for posterity: https://github.com/square/wire/blob/master/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WirePlugin.kt#L114-L116 (`protoLIbrary = true` is considered a target)